### ISSUE TITLE
bridges testing - assignment numbers..

### ISSUE THREAD

**Issue by krs-world** [krs-world/Bridges_Testing#4]
Lets reserve assignment numbers 0-99 for C++ tests, 100-199 for Java tests and 200-299 for Python tests.

I made some changes to the Makefiles (top level and the C++ and Java makefiles), so do a pull prior to any changes.

The top level Makefile should now run both the Java and C++ tests, Python has been commented out till it is ready.
   
       -- krs

**Comment by krs-world** [comment]
I have updated the bridges testing repo - added 7 Java tests that uses real-world data.  Next need to do the C++ version. Matthew, please do the python equivalent for these.

**Comment by krs-world** [comment]
Ok, I have now updated the C++ and Java tests. There are two sets of tests for each language:

Basic tests for each data structure (15 tests)
Data source dependent (8 tests)

Some of these are from the workshop and some from my earlier tests I had collected.

The numbering is as follows:

Java 

0-14 -- basic data structure tests
100-107 - data source dependent tests

C++
50-64 - basic data structure tests
200-207 -  data source dependent tests

Hmm, didn think about python:

Python could be 

75-89 - basic data structure tests
300-307 - data source dependent tests. 

Perhaps in hte next version, we can put the assignment number in the Makefile and send it as command line argument.

        -- krs
